libvmaf/src/feature/cambi.c

```diff
@@ -719,9 +734,34 @@ static FORCE_INLINE inline double weight_scores_per_scale(double *scores_per_sca
     return score / normalization;
 }
 
-static int cambi_score(VmafPicture *pics, uint32_t *mask_dp, uint16_t window_size, double topk,
-                       const uint16_t *tvi_for_diff, float *c_values, uint16_t *c_values_histograms, 
-                       uint8_t *filter_mode_histogram, uint16_t *filter_mode_buffer, double *score) {
+static void write_uint16(FILE *file, uint16_t v) {
+    fwrite((void*)(&v), sizeof(v), 1, file);
+}
+
+static void dump_c_values(const float *c_values, int width, int height, int scale, int window_size, const char *dir_path) {
+    int max_diff_weight = g_diffs_weights[0];
+    for (int i = 0; i < NUM_DIFFS; i++) {
+        if (g_diffs_weights[i] > max_diff_weight) {
+            max_diff_weight = g_diffs_weights[i];
+        }
+    }
+    char path[1024];
+    snprintf(path, sizeof(path), "%s/cambi_heatmap_scale_%d_%dx%d_16b.gray", dir_path, scale, width, height);
```

Comment:
"/" is probably not a cross-platform.

Reply:
I have seen some [posts](https://stackoverflow.com/questions/12971499/how-to-get-the-file-separator-symbol-in-standard-c-c-or/30261289) suggesting something like this to set up the separator:
```
const char separator = 
#ifdef _WIN32
    '\\'
#else
    '/'
#endif
;
```

There are a couple references to `_WIN32` in this repo, but none of them is used to determine the separator in this way. Do you think this is fine?